sanssouci/tests/test_post_hoc_bounds.py

```diff
@@ -145,3 +146,40 @@ def test_curve_max_fp():
     assert len(curve_max_fp_) == len(p_values)
     assert all(curve_max_fp_ >= 0)
     assert curve_max_fp_[-1] == max_fp(p_values, thresholds)
+
+
+def test_curve_min_tdp():
```

Comment:
Also test consistency between curve_min_tdp() and curve_max_fp(), using the fact that min_tdp(S) = 1 - max_fp(S)/|S|